Bio/MarkovModel.py

```diff
@@ -75,14 +81,15 @@ def __str__(self):
 
 
 def _readline_and_check_start(handle, start):
+    """Read the first line and evaluate that begisn with the correct start (PRIVATE)."""
     line = handle.readline()
     if not line.startswith(start):
         raise ValueError("I expected %r but got %r" % (start, line))
     return line
 
 
 def load(handle):
-    """load(handle) -> MarkovModel()"""
+    """Load(handle) -> MarkovModel()."""
```

Comment:
This is written like a function signature, not human readable. How about: ``"""Parse a file handle into a MarkovModel object."""``